### PR TITLE
Upgrade git, dep and operator-sdk

### PR DESCRIPTION
**Pull request by sthaha**

### CODE REVIEW

File: openshift-ci/Dockerfile.tools [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7 as build-tools
+FROM centos:8 as build-tools
```
Comment (chmouel) [inline openshift-ci/Dockerfile.tools]: if you use centos:8 why do you need to update git manually?

Comment (sthaha) [reply openshift-ci/Dockerfile.tools]: where did I update git "manually" ?

Comment (sthaha) [reply openshift-ci/Dockerfile.tools]: the `tools` image will have an updated git is what I meant

Comment (nikhil-thomas) [reply openshift-ci/Dockerfile.tools]: height of best practices in git commit messages :P

**Comment by nikhil-thomas** [comment]
force merging this patch as openshift/release build-root take the `dockerfile.tools` from the current master not from the patch

```
Pulling image centos:7 ...                                                                                                                        
--> FROM centos:7 as build-tools                                                                                                                  
--> LABEL maintainer="pipelines-dev <pipelines-dev@redhat.com>"   
...
```
base image should be centos:8

**Pull request merged**